packages/apollo-server-express/src/__tests__/expressApollo.test.ts

```diff
@@ -1,5 +1,6 @@
 import express from 'express';
 import { ApolloServer, ApolloServerExpressConfig } from '../ApolloServer';
+import { graphqlExpress } from '../expressApollo'
```

Comment:
It was the lack of trailing semi-colon here which was the problem since that doesn't align with Prettier, and this project's, configured patterns.